clientd3d/object.h

```diff
@@ -79,6 +79,7 @@ typedef struct {
 		                     this field gives amount of object */
    DWORD     temp_amount;         /* Scratch field used when user is selecting amount of object */
    int       flags;               /* Flags describing various properties of objects */
+   int       text_color_type;     /* Item text color type */
```

Comment:
This comment is not exactly illuminating.  What is a text color type?

Reply:
updated object member to be `rarity` and updated comment